lldb/test/API/functionalities/progress_reporting/swift_progress_reporting/TestSwiftProgressReporting.py

```diff
@@ -8,26 +8,28 @@
 from lldbsuite.test.lldbtest import *
 from lldbsuite.test.decorators import *
 
-class TestSwiftProgressReporting(TestBase):
 
+class TestSwiftProgressReporting(TestBase):
     mydir = TestBase.compute_mydir(__file__)
 
     def setUp(self):
         TestBase.setUp(self)
         self.broadcaster = self.dbg.GetBroadcaster()
-        self.listener = lldbutil.start_listening_from(self.broadcaster,
-                                        lldb.SBDebugger.eBroadcastBitProgress)
+        self.listener = lldbutil.start_listening_from(
```

Comment:
Is this your own reformatting or `black` output ? If it's the former, we should avoid moving code around for nothing as it rewrites history.

Reply:
This was from `black`'s output after writing to the file, I can change it back to its original formatting if needed.